src/fixtures/configs/rows-without-name.json

```diff
@@ -0,0 +1,43 @@
+{
+  "componentId": "transformation",
+  "name": "legacy transformation",
+  "configuration": {
+    "parameters": {
+      "foo": "bar"
+    }
+  },
+  "rows": [
```

Comment:
Mame tu 5 testovacich config rows:
- prve 2 nemaju nastavene `name`, ale maju `name` v koniguracii
  - to simuluje legacy transformacie, ktore to takto niekedy mali, a este sa mozu v KBC vyskytnut
- dalsie 3 nemaju `name` nastavene vobec 
  - tento stav je v API validny, aj ked sa v UI neda dosiahnut

Pouzivaju sa v integracnom teste `pull-rows-without name`